UPGRADE-3.x.md

```diff
@@ -1,6 +1,20 @@
 UPGRADE 3.x
 ===========
 
+UPGRADE FROM 3.6 to 3.7
+=======================
```

Comment:
Please remove this title.

Reply:
Woops! Done.

Reply:
is it cached or is the title still available @greg0ire 👀

Reply:
I must have pushed it again from work or home by accident :P